charts/portworx/templates/portworx-controller.yaml

```diff
@@ -104,7 +104,9 @@ spec:
       - command:
         - kube-controller-manager
         - --leader-elect=true
+        {{- if (semverCompare "<1.22" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
Doesn't this require to strip out only the version part like you did for stork scheduler version?
```
.Capabilities.KubeVersion.GitVersion | regexFind "^v\\d+\\.\\d+\\.\\d+
```

Reply:
updated :)